### PR TITLE
Svg icons

### PR DESCRIPTION
**Pull request by TobiTenno**
### Warframe Hub Pull Request
---
---

**Summary (short):** Using svgs for resources where possible. See deployment for how it looks

(Check one)
- [ ] Issue fix
- [x] Suggestion fulfillment

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -167,6 +167,11 @@ img {
 /* Nightmare glyph */
 .nightmare, .archwing, .fissureGlyph {
   margin-top: -3px;
+}
+
+.nightmare:not([src*="/svg/"],
```
Comment (Tithen-Firion) [inline assets/css/main.css]: I think we should keep it simple, just invert image colors so png matches svg.

Comment (TobiTenno) [reply assets/css/main.css]: like make all the pngs black?

Comment (TobiTenno) [reply assets/css/main.css]: i'll do a pass tonight when i get home

**Review by EricSihaoLin — APPROVED** [review]

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**